RELEASE_NOTES.md

```diff
@@ -59,6 +60,21 @@ The following changes are relevant for v4 custom configs that replaced certain f
   - `/ldp/modes/default.json`
 - The `PermissionReader` structure has changed to be more consistent.
   - `/ldp/authorization/*`
+- Updated template configs.
+  - `/app/main/general/templates.json` was added to configure a generic template engine handler.
+  - `/app/main/default.json` now imports the above config file.
+  - Files configuring template engines: 
+    - `/app/init/initializers/prefilled-root.json`
+    - `/app/init/initializers/root.json`
+    - `/app/setup/handlers/setup.json`
+    - `/identity/access/initializers/idp.json`
+    - `/identity/access/initializers/well-known.json`
+    - `/identity/handler/interaction/routes/forgotten-password.json`
+    - `/identity/handler/interaction/views/html.json`
+    - `/identity/pod/resource-generators/templated.json`
+    - `/util/representation-conversion/convertors/dynamic-json-template.json`
+    - `/util/representation-conversion/convertors/errors.json`
+    - `/util/representation-conversion/convertors/markdown.json`
```

Comment:
Let's just replace this with "All configuration files that instantiate a `TemplateEngine`" 😅